src/net/KNet/ClientSide/BridgedClasses/Clients/Consumer/KNetConsumer.cs

```diff
@@ -0,0 +1,132 @@
+﻿/*
+*  Copyright 2022 MASES s.r.l.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+*  Refer to LICENSE for more information.
+*/
+
+using MASES.KNet.Common;
+using MASES.KNet.Common.Serialization;
+using Java.Time;
+using Java.Util;
+using Java.Util.Regex;
+using MASES.JCOBridge.C2JBridge;
+using System;
+
+namespace MASES.KNet.Clients.Consumer
+{
+    public class Message<K, V>
+    {
+        readonly KNetConsumerCallback<K, V> obj;
+        internal Message(KNetConsumerCallback<K, V> obj)
+        {
+            this.obj = obj;
+        }
+
+        //public string Topic => data.TypedEventData;
+
+        //public int Partition => data.GetAt<int>(0);
```

Comment:
Why?

Reply:
Same of https://github.com/masesgroup/KNet/pull/55#discussion_r861829072. We leaved the code of a test made.